src/vs/base/browser/ui/iconLabel/iconLabelHover.ts

```diff
@@ -127,18 +135,28 @@ export function setupCustomHover(hoverDelegate: IHoverDelegate, htmlElement: HTM
 		});
 	};
 	const mouseOverDomEmitter = dom.addDisposableListener(htmlElement, dom.EventType.MOUSE_OVER, () => showHoverDelayed(hoverDelegate.delay), true);
-	return {
+	const hover: ICustomHover = {
 		show: () => {
 			showHoverDelayed(0); // show hover immediately
 		},
 		hide: () => {
 			hideHover(true, true);
 		},
+		update: newTooltip => {
+			markdownTooltip = newTooltip;
```

Comment:
It is a bit ugly to assign to the parameters of the method, let me know if you have a different suggestion.

Reply:
We could move make tooltip a parameter of `showHover`
-> 
`new RunOnceScheduler(() => getTooltipForCustom(markdownTooltip), delay);